templates/password.html

```diff
@@ -0,0 +1,46 @@
+{{ template "header.html" . }}
+
+<div class="card w-75 mt-5 shadow-sm">
+  <div class="card-body">
+    <div class="row user-select-none">
+      <picture class="col-lg-6">
+        <img class="img d-block w-auto-sm w-50-md" src="{{ "logoImage" | extra }}" alt="{{ "logoImageAlt" | extra }}">
```

Comment:
This should have defaults set so that if logoImage is not defined there is something presented.  This should all work with zero configuration.